SPECS/cereal/cereal.spec

```diff
@@ -0,0 +1,129 @@
+# Debuginfo packages are disabled to prevent rpmbuild from generating an empty
+# debuginfo package for the empty main package.
+%global debug_package %{nil}
+
+Vendor:         Microsoft Corporation
```

Comment:
File could use some light linting, but looks good overall!

Reply:
Absolutely. Thank you